dags/DAG_url_processor.py

```diff
@@ -52,15 +53,24 @@ def extract_data(url):
         }
 
 
+def create_dynamic_dag(dag_obj, language):
+    with dag_obj:
+        processor = PythonOperator(task_id='de_url_processor_operator',
+                                   python_callable=url_processor,
+                                   op_kwargs={'language': language})
+
+    return dag_obj
+
+
 default_args["start_date"] = datetime.datetime(2020, 9, 1, 0, 0, 0)
-dag = DAG('de_url_processor',
-          schedule_interval='* * * * *',
-          description='Scrape website for newspaper',
-          default_args=default_args,
-          catchup=False,
-          )
-
-with dag:
-    processor = PythonOperator(task_id='de_url_processor_operator',
-                               python_callable=url_processor,
-                               op_kwargs={'language': 'de'})
+
+LANGUAGES = ['de', 'tr']
+for language in LANGUAGES:
```

Comment:
This is almost identical to the previous setup. Note that both DAGs should have a starting date that is different by 30 seconds such that they are not scheduled at the same time.

Reply:
Updated